src/tiled/mapdocument.cpp

```diff
@@ -125,6 +127,58 @@ MapDocument::~MapDocument()
     delete mMap;
 }
 
+/**
+ * Currently, this is the entry point for testing templates,
+ * it creates a templateGroup from the selected objects except the last object,
+ * then creates a templatGroupDocument from the templateGroup,
```

Comment:
Typo: templatGroupDocument (also, I think it would be better to match the casing with the class name).